modules/installation/pages/network-requirements.adoc

```diff
@@ -4,7 +4,7 @@
 This section details the networking and port requirements for {productname}.
 
 Fully Qualified Domain Name (FQDN)::
-The {productname} server must resolve its FQDN correctly or cookies will not work properly on the WebUI.
+The {productname} server must resolve its FQDN correctly. This is a hard requirement else various components might malfunction.
```

Comment:
```suggestion
The {productname} server must resolve its FQDN correctly. 
If the FQDN cannot be resolved, it can cause serious problems in a number of different components.
```